src/es/components/organisms/form/Form.js

```diff
@@ -0,0 +1,38 @@
+// @ts-check
+import { Shadow } from '../../web-components-toolbox/src/es/components/prototypes/Shadow.js'
+
+/**
+* @export
+* @class Form
+* @type {CustomElementConstructor}
+*/
+export default class Form extends Shadow() {
```

Comment:
rather use the [SimpleForm ](https://github.com/mits-gossau/web-components-toolbox/tree/master/src/es/components/molecules/simpleForm) or [SimpleFormValidation](https://github.com/mits-gossau/web-components-toolbox/tree/master/src/es/components/molecules/simpleFormValidation) and make a new theme.